staging/src/k8s.io/apimachinery/pkg/api/meta/meta.go

```diff
@@ -375,6 +375,23 @@ func (resourceAccessor) SetContinue(obj runtime.Object, version string) error {
 	return nil
 }
 
+func (resourceAccessor) ClusterName(obj runtime.Object) (string, error) {
```

Comment:
This isn't necessary in this PR - I was going to use it for something, but didn't end up needing it. I can back it out, or we can keep it.